plugins/jobs/src/js/components/Breadcrumbs.js

```diff
@@ -6,57 +6,66 @@ import Breadcrumb from "#SRC/js/components/Breadcrumb";
 import BreadcrumbTextContent from "#SRC/js/components/BreadcrumbTextContent";
 import PageHeaderBreadcrumbs from "#SRC/js/components/PageHeaderBreadcrumbs";
 
-export default function Breadcrumbs({ item, children, states }) {
-  function getBreadcrumb(item, id = "", name = "") {
-    const isDetailPage = [...item.path, item.name].join(".") === id;
-    const link = isDetailPage ? `/jobs/detail/${id}` : `/jobs/overview/${id}`;
-
-    return (
-      <Breadcrumb key={id} title="Jobs">
-        <BreadcrumbTextContent>
-          <Link to={link}>{name === "" ? "Jobs" : name}</Link>
-        </BreadcrumbTextContent>
-
-        {isDetailPage ? states : null}
-      </Breadcrumb>
-    );
-  }
-
-  function getBreadcrumbList(item) {
-    if (item == null) {
-      return [];
-    }
+/**
+ * @param  {string} link
+ * @param  {string[]} pathParts path including name (if given)
```

Comment:
Can we change this comment? I initially thought that pathParts are optional, whereas only the name is optional.

Can we phrase it like: "path of names, empty means overview page"